bot.js

```diff
@@ -38,15 +39,19 @@ class Bot {
             return;
         }
 
-        if (config.DEBUG) this.log("Starting ", cmd.join(' '));
-
         this.proc.stderr.pipe(split2()).on('data', (data) => {
             if (this.ignore)  return;
             const errline = data.toString().trim();
             if (errline === "") return;
             this.error(`stderr: ${errline}`);
 
             if (config.ogspv) this.pv.postPvToChat(errline);
+            if (config.aichat) {
+               var chat_match = /(DISCUSSION|MALKOVICH):(.*)/.exec(errline)
```

Comment:
also we dont want to use `var`, prefer `let` or even better `const` if possible
https://github.com/online-go/gtp2ogs/pull/241/files#diff-9d199097bf1be7e582b7cb3a378cfbf0R50

Reply:
fixed